ThreeDSecure/src/main/java/com/braintreepayments/api/ThreeDSecureClient.java

```diff
@@ -285,48 +264,54 @@ public void tokenize(ThreeDSecurePaymentAuthResult paymentAuthResult,
                             (threeDSecureResult1, error) -> {
                                 if (threeDSecureResult1 != null) {
                                     if (threeDSecureResult1.hasError()) {
-                                        braintreeClient.sendAnalyticsEvent(
-                                                "three-d-secure.verification-flow.upgrade-payment-method.failure.returned-lookup-nonce");
-                                        callback.onThreeDSecureResult(new ThreeDSecureResult.Failure(new BraintreeException(threeDSecureResult1.getErrorMessage()), threeDSecureResult1.getThreeDSecureNonce()));
+                                        braintreeClient.sendAnalyticsEvent(ThreeDSecureAnalytics.JWT_AUTH_FAILED);
+                                        callbackTokenizeFailure(callback, new ThreeDSecureResult.Failure(new BraintreeException(threeDSecureResult1.getErrorMessage()), threeDSecureResult1.getThreeDSecureNonce()));
                                     } else if (threeDSecureResult1.getThreeDSecureNonce() != null) {
-                                        braintreeClient.sendAnalyticsEvent(
-                                                "three-d-secure.verification-flow.upgrade-payment-method.succeeded");
-                                        sendLiabilityShiftedAnalytics(threeDSecureResult1);
-                                        callback.onThreeDSecureResult(new ThreeDSecureResult.Success(threeDSecureResult1.getThreeDSecureNonce()));
+                                        braintreeClient.sendAnalyticsEvent(ThreeDSecureAnalytics.JWT_AUTH_SUCCEEDED);
+                                        callbackTokenizeSuccess(callback, new ThreeDSecureResult.Success(threeDSecureResult1.getThreeDSecureNonce()));
                                     }
                                 } else if (error != null) {
-                                    braintreeClient.sendAnalyticsEvent(
-                                            "three-d-secure.verification-flow.upgrade-payment-method.errored");
-                                    callback.onThreeDSecureResult(new ThreeDSecureResult.Failure(error, null));
+                                    braintreeClient.sendAnalyticsEvent(ThreeDSecureAnalytics.JWT_AUTH_FAILED);
```

Comment:
Similar to my comment on the other analytics PR - would we want to send the failure type as a param instead?

Reply:
resolving - similar to this thread: https://github.com/braintree/braintree_android/pull/920#discussion_r1488551108

Reply:
Resolved in other PR - keeping these events to make it easier to see what extra info should be added to each of these events - they can be removed when the extra params are added to analytics